src/styles/reset.scss

```diff
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+}
+
+p {
+  margin: 0;
+}
```

Comment:
Create variables for colors, which appear many times in styles files

Reply:
To make better choices you can use this guide https://mate-academy.github.io/style-guides/htmlcss.html